Aquality.Selenium.Template/Aquality.Selenium.Template/Implementation/CustomStartUp.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Aquality.Selenium.Elements;
+using Aquality.Selenium.Elements.Interfaces;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using Aquality.Selenium.Core.Localization;
+using Aquality.Selenium.Core.Logging;
+using System.Reflection;
+using Aquality.Selenium.Core.Applications;
+using Aquality.Selenium.Configurations;
+using Aquality.Selenium.Core.Configurations;
+using ICoreElementFactory = Aquality.Selenium.Core.Elements.Interfaces.IElementFactory;
+using ICoreTimeoutConfiguration = Aquality.Selenium.Core.Configurations.ITimeoutConfiguration;
+using ILoggerConfiguration = Aquality.Selenium.Core.Configurations.ILoggerConfiguration;
+using ITimeoutConfiguration = Aquality.Selenium.Configurations.ITimeoutConfiguration;
+using TimeoutConfiguration = Aquality.Selenium.Configurations.TimeoutConfiguration;
+using Aquality.Selenium.Browsers;
+using Aquality.Selenium.Template.Interfaces;
+
+namespace Aquality.Selenium.Template.Implementation
+{
+    public class CustomStartUp : BrowserStartup
+    {
+        public override IServiceCollection ConfigureServices(IServiceCollection services, Func<IServiceProvider, IApplication> applicationProvider,
+            ISettingsFile settings = null)
+        {
+            settings = settings ?? GetSettings();
+            base.ConfigureServices(services, applicationProvider, settings);
+            services.AddTransient<IElementFactory, ElementFactory>();
+            services.AddTransient<ICoreElementFactory, ElementFactory>();
+            services.AddSingleton<ICustomTimeoutConfiguration>(serviceProvider => new CustomTimeoutConfiguration(settings));
```

Comment:
discussed, leave only needed dependencies

Reply:
The CustomStartUp was updated.